### ISSUE TITLE
Initital Setup

### ISSUE THREAD

**Issue by bcoopervcom** [CMU-Perceptual-Computing-Lab/openpose#227]
System Configuration
Windows 10 
Visual Studio 2015
Cuda 8.0
cuDNN 8.0
Graphic card Nvidia GeForce GTX 1070

We are a research team and have very little knowledge on installing software.  
### Issue summary
We are attempting to install the OpenPose software once we get to step 4 of installation and run openpose.sln Microsoft's visual studio says unable to start program C:\openpose-master\windows\x64\release\openpose.dll.  It says the file is not a valid Win32 application. 

### Executed command (if any)
When we attempt to run the openposedemo.exe from command we receive.  F0828 13:19:44.212276 3052 io.cpp:41 Check failed: fd != -1 (-1 vs. -1) File not found: models/pose/coco/pose_deploy_linevec.protxt.  Checke failure stack trace:

### Type of issue
You might select multiple topics, delete the rest:
- Compilation/installation error
- Execution error
- Help wanted
- Question
- Enhancement / offering possible extensions / pull request / etc
- Other (type your own type)

**Comment by gineshidalgo99** [comment]
If it is not finding the prototxt file is because the installation steps were not correctly followed. Please, try again with the latest OpenPose version. Thanks

**Comment by nellyneil** [comment]
Hi! I'm a member of another research team in exactly the same situation. The only difference is that I'm running Windows 8.1 and I do have the prototxt file. I've tried following the OpenPose installation steps several times in the past and I'm giving it another go. To my knowledge, I've followed the installation steps as closely as possible. I launched with CMake GUI and was able to properly successfully generate and open the project. However, I keep receiving the same error as this original poster: that openpose.dll could not be found. Is there any way to fix this?

Best,
Cornelius
